src/components/timeline/TimelineViewControls.svelte

```diff
@@ -135,10 +150,24 @@
 >
   <RotateCounterClockwiseIcon />
 </button>
+<button
+  class="st-button icon"
+  on:click={onToggleDirectiveVisibility}
+  use:tooltip={{
+    content: `${allDirectivesVisible ? 'Hide' : 'Show'} Directives on all Timeline Rows`,
+    placement: 'bottom',
```

Comment:
Could we do top to match the edit timeline button?

Reply:
> Could we do top to match the edit timeline button?

but don't all the tooltips in this component have a `bottom` placement?

Reply:
Ah yes, bottom on the global controls, top on the row controls..